### DISCUSSION TITLE
search in content. what are known filetypes?

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2412] [Q&A] [answered]
* v 2.11.1
* i am unable to search text inside .ris file
* str
  * dir._ internal storage_ dir1
  * search "random text"
  * tick "search in content"
* github doesnt allow attaching .ris, so changed attached filetype to .txt

[w,22-4,14-5,23.txt](https://github.com/user-attachments/files/16980050/w.22-4.14-5.23.txt)

**Comment by harshad1** [comment]
We test if a file is a text file before attempting to search it's contents.

This format may be failing a mime type test

**Comment by gsantner** [comment, marked as answer]
It might be not detectable as textfile by the system like harshad1 said. Never heard of this file/mimetype either.

If needed and most likely case that .ris is a textfile.. changes can be added to Markor to manually remap .ris to a `text/` mime type.

**Reply by ghost** [reply]
@gsantner 
_Never heard of this file/mimetype either._
* Never heard of this file/mimetype either.
  * RIS is a standardized tag format developed by Research Information Systems, Incorporated (the format name refers to the company) to enable citation programs to exchange data.
  * https://en.wikipedia.org/wiki/RIS_(file_format)

**Comment by harshad1** [comment]
There is an option to specify extensions for opening in markor. Try adding ris there?
